.github/ISSUE_TEMPLATE/1-bug-report.yml

```diff
@@ -0,0 +1,42 @@
+name: Bug report
+description: File a bug report to help us improve
+labels: ["bug"]
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what the bug is.
+    validations:
+      required: true
+  - type: textarea
+    id: how-to-reproduce
+    attributes:
+      label: To reproduce
+      description: Please describe the steps to reproduce the behavior
+      placeholder: |
+        1. Include code snippet as short as possible
+        2. Specify the command which should be used to compile the program
+        3. Specify the comment which should be used to launch the program
+        4. Indicate what is wrong and what was expected
+    validations:
+      required: false
+  - type: textarea
+    id: environment
+    attributes:
+      label: Environment
+      description: Please complete the following information
+      placeholder: |
+        - OS: [e.g Windows/Linux]
+        - Target device and vendor: [e.g. Intel GPU]
+        - DPC++ version: [e.g. commit hash or output of `clang++ --version`]
+        - Dependencies version: [e.g. low-level runtime versions (like NEO 20.04)]
```

Comment:
Maybe instead ask customers to provide the output of `sycl-ls --version` command?